packages/fx-core/templates/plugins/resource/bot/bicep/provision.template.v2.bicep

```diff
@@ -3,7 +3,7 @@ module botProvision '{{PluginOutput.fx-resource-bot.Modules.botProvision.Provisi
   name: 'botProvision'
   params: {
     provisionParameters: provisionParameters
-    userAssignedIdentityId: {{PluginOutput.fx-resource-identity.References.identityResourceId}}
+    userAssignedIdentityId: {{PluginOutput.fx-resource-identity.References.identityClientId}}
```

Comment:
here should be identityResourceId not identityClientId

Reply:
fixed